terraform/templates/ec2_user_data.sh

```diff
@@ -34,6 +34,21 @@ cat > /etc/profile.d/libra_prompt.sh <<EOF
 export PS1="[\u:validator@\h \w]$ "
 EOF
 
+if ${enable_logrotate}; then
+	cat >> /etc/logrotate.conf <<"EOF"
+	${log_path} {
+		daily
+		rotate 100
+		size 10M
```

Comment:
Can you give some details on what this config mean?

Does it mean it going to rotate when size reaches 10M? If yes , thats too small for things like cluster test, it will be rotating files constantly

Reply:
@andll Yeah, it means it will rotate when size reaches 10M, if it didn't reach, it will rotate daily. And it will keep 100 rotated files.
What would be a reasonable size to rotate? 100M?

Reply:
Can we just remove rotation on size and only rotate once a day?
For prod this should be ok(don't anticipate too many logs as they going to run with RUST_LOG=info), and for cluster test I would prefer this to be predictable

Reply:
@andll Sure, sounds good to me.